README.md

```diff
@@ -30,7 +28,7 @@ dependency to your [composer.json][composer].
 ```json
 {
 	"require": {
-		"onoi/http-request": "~1.3"
+		"onoi/http-request": "~2.0"
```

Comment:
Since all the SMW packages are under `mediawiki/semantic-...`, is this package going to be `mediawiki/http-request`?